### PR TITLE
Doc/improve

### PR DESCRIPTION
**Pull request by HansRobo**
## Types of PR

- [ ] Documentation

### CODE REVIEW

**Comment by HansRobo** [comment]
- Replace "Tier IV" with "TIER IV"
- Add workflow to check broken links
- Fix broken links
- Add update date to all documents
- Fix typos
- Delete old information (WIP. I will fix it in the next PR)

**Comment by HansRobo** [comment]
There is a conflict in `.github/workflows/Documentation.yaml`.
But I cannot resolve it because I don't have a write permission to master branch.
Could you resolve it?
Please overwrite master with this PR.

File: external/lanelet2_extension_psim/docs/lanelet2_format_extension.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ About the basics of the default format, please refer to main [Lanelet2 repositor
 In addition to default Lanelet2 Format, users should add following mandatory/optional tags to their osm lanelet files as explained in reset of this document. 
 Users may use `autoware_lanelet2_validation` [node](../README.md#nodes) to check if their maps are valid. 
 
-There are also [extra regulatory elements](./ExtraRegulatoryElements) added for Autoware:
+There are also [extra regulatory elements](./extra regulatory elements.md) added for Autoware:
```
Comment (yamacir-kit) [inline external/lanelet2_extension_psim/docs/lanelet2_format_extension.md:8]: Is this modification intentional?

**Review by yamacir-kit — CHANGES_REQUESTED** [review]

File: docs/design/About.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ This framework is designed for executing scenario tests with Autoware.
 
 ## Simple sensor simulator
 
-If you want to know about our very very simple lidar simulator, please read [this documentation](SimpleSensorSimulator.md).
+If you want to know about our very, very simple lidar simulator, please read [this documentation](SimpleSensorSimulator.md).
```
[the remark this answers is not in the document]
Comment (HansRobo) [reply docs/design/About.md:13]: Yes.
It seems more formal to put a comma between very and very.
For example, this notation is used in the New York Times.

File: external/lanelet2_extension_psim/docs/lanelet2_format_extension.md (hunk 1 above)
Comment (HansRobo) [reply external/lanelet2_extension_psim/docs/lanelet2_format_extension.md:8]: Yes.
Both writing methods work, but the writing method before the change gives an error in the CI of LinkCheck.

File: docs/design/BehaviorPlugin.md [hunk 3]
```diff
@@ -123,7 +123,7 @@ Setters are named as set(Foo), such as `BehaviorPluginClass::setCurrentTime(doub
 | GoalPoses            | Goal poses of entity.                                 | `std::vector<geometry_msgs::msg::Pose>`                      |
 | HdMapUtils           | Shared pointer of HdMapUtils class.                   | `std::shared_ptr<hdmap_utils::HdMapUtils>`                   |
 | Obstacle             | Target obstacle of your NPC.                          | `boost::optional<traffic_simulator_msgs::msg::Obstacle>`     |
-| OtherEntityStatus    | Dictionay of other entity status.                     | `EntityStatusDict`                                           |
+| OtherEntityStatus    | Dictionary of other entity status.                     | `EntityStatusDict`                                           |
```
[the remark this answers is not in the document]
Comment (HansRobo) [reply docs/design/BehaviorPlugin.md]: Fixed in 8f600eb880db50927158d05dc6780344af90c9fc

File: docs/design/SimpleSensorSimulator.md [hunk 4]
```diff
@@ -3,17 +3,18 @@
...
 
 The simple sensor simulator is a reference implementation of the simulator which follows our scenario testing framework.
-This package includes very very simple lidar simulation and send simulated detection result to the Autoware.
+This package includes very, very simple lidar simulation and send simulated detection result to the Autoware.
 
 <font color="#065479E">_Note! Simple Sensor Simulator is just a reference implementation, so we can adapt any kinds of autonomous driving simulators if we can develop ZeroMQ interface to your simulator._</font>
 
 In lidar simulation, we use intel's ray-casting library embree.
 
+[//]: # (TODO : Fix invalid embeded frame)
```
[the remark this answers is not in the document]
Comment (HansRobo) [reply docs/design/SimpleSensorSimulator.md]: Deleted in 44b5608dc64cf3ae566c4a7ff29e89bfd36ca5c0

**Review by yamacir-kit — APPROVED** [review]

**Pull request merged**